test/analysis_options_test.dart

```diff
@@ -68,7 +68,7 @@ void main() {
     final fileName = file.path.split(Platform.pathSeparator).last;
```

Comment:
This is now unused, correct? Maybe we should delete it?

Reply:
It's used as the test name on line 70

Reply:
🤕  I missed that.